.coderabbit.yaml

```
@@ -0,0 +1,190 @@
+  related_prs: true
+  suggested_labels: true
+  auto_apply_labels: false
+  suggested_reviewers: true
+  auto_assign_reviewers: false
+  poem: false
+  labeling_instructions:
+    - label: ""
+      instructions: ""
+  path_filters: ['!src/ignoreFolder']
+  path_instructions:
+  - path: "*"
```

YAMLlint (1.37.1)

[warning] wrong indentation: expected 4 but found 2

(indentation)